featuretools/computational_backends/calculate_feature_matrix.py

```diff
@@ -141,7 +141,8 @@ def calculate_feature_matrix(features, entityset=None, cutoff_time=None, instanc
         cutoff_time = pd.DataFrame(map_args, columns=['instance_id', 'time'])
     else:
         cutoff_time = cutoff_time.reset_index(drop=True)
-
+        assert (cutoff_time.duplicated().sum() == 0), \
```

Comment:
Let's make this check more specific. it should check if the instance id and time are duplicated regardless of the pass through columns being different.